o2a/o2a_libs/el_wf_functions.py

```diff
@@ -48,6 +48,22 @@ def conf(key: str) -> str:
     return "params[{}]".format(key)
 
 
+@contextfunction
+def user(context=None) -> str:
+    """
+    Return DAG owner or raises error if there is more than one owner.
+    """
+    dag: Optional[DAG] = context.get("dag", None)
+    if dag is None:
+        raise AirflowException("No dag reference in context.")
+
+    owners: List[str] = dag.owner.split(",")
```

Comment:
```suggestion
    owners: Set[str] = {t.owner for t in dag.tasks}
```
https://github.com/apache/airflow/blob/master/airflow/models/dag.py#L558
This will allow the use of a comma in the name of the owner.